ext/odbc/php_odbc.c

```diff
@@ -970,7 +962,6 @@ PHP_FUNCTION(odbc_prepare)
 	} else {
 		result->values = NULL;
 	}
-	Z_ADDREF_P(pv_conn);
```

Comment:
as far as I saw, this was added many years ago (along with another similar `Z_ADDREF_P` below) I'm not sure exactly why. However, these caused many leaks which were fixed by their removal..